src/index.js

```diff
@@ -1,7 +1,5 @@
-
 import './style.css';
 
-
 const allTasks = document.querySelector('.task-list');
 
 const listArray = [
```

Comment:
- [ ] All the changes to the project are not reflected in the pull request. Let's make sure you have all the changes here.